### PR TITLE
TEST - Add passthroughcopy option to eleventy

### PR DESCRIPTION
**Pull request by devnook**

### CODE REVIEW

File: .eleventy.js [hunk 1]
```diff
@@ -257,6 +258,24 @@ module.exports = function (config) {
...
+    'advanced-apps/badges',
+    // Add more pattern ids on as-needed basis.
+  ];
+  standaloneJSCodePatterns.forEach((pattern) => {
+    const src = path.join('src/site/content/en/patterns/', pattern, 'assets');
+    // Target file rewrite, e.g.: 'src/site/content/en/patterns/advanced-apps/badges/assets/*.js': 'patterns/advanced-apps/badges'
+    const rewrite = {};
+    rewrite[path.join(src, '*.js')] = path.join('patterns', pattern);
+    rewrite[path.join(src, '*.json')] = path.join('patterns', pattern);
+    rewrite[path.join(src, '*.svg')] = path.join('patterns', pattern);
```
Comment (tomayac) [inline .eleventy.js:275]:
```suggestion
    rewrite[path.join(src, '*.svg')] = path.join('patterns', pattern);
    rewrite[path.join(src, '*.png')] = path.join('patterns', pattern);
```

**Comment by devnook** [comment]
Current prod staging: https://web-dev-production-1--staging-duuagiud.web.app/patterns/advanced-apps/badges/demo.html

**Comment by tomayac** [comment]
> Current prod staging: https://web-dev-production-1--staging-duuagiud.web.app/patterns/advanced-apps/badges/demo.html

Looks good, thanks! Now we just need the `.png` and `.svg` files.

**Comment by devnook** [comment]
Closing in favor of https://github.com/GoogleChrome/web.dev/pull/7875

**Pull request closed without merging**